comment_iowriter.go

```diff
@@ -142,3 +144,135 @@ func (cw *RDJSONCommentWriter) Flush(_ context.Context) error {
 	}
 	return nil
 }
+
+var _ CommentService = &SARIFCommentWriter{}
+
+// SARIFCommentWriter
+type SARIFCommentWriter struct {
+	w        io.Writer
+	comments []*Comment
+	toolName string
+}
+
+func NewSARIFCommentWriter(w io.Writer, toolName string) *SARIFCommentWriter {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported function NewSARIFCommentWriter should have comment or be unexported
<!-- __reviewdog__:ChA3NjAyNTQyOTdlYmUyYjUyEg9nb2xpbnQtY2lyY2xlY2k= -->